common/src/graph.rs

```diff
@@ -16,16 +15,18 @@ use crate::{
 /// Graph of the right dodecahedral tiling of H^3
 #[derive(Debug, Clone)]
 pub struct Graph<N> {
-    nodes: Vec<Node<N>>,
+    nodes: FxHashMap<NodeId, Node<N>>,
```

Comment:
Because `NodeId` is already uniformly distributed, we could use an identity hasher to make access even cheaper. Probably won't matter, but it'd be cute.

Reply:
Oh yeah, I had considered that but didn't know how to set up such a hasher (or what it was called). It looks like it would need a new crate or a manual implementation, so it probably makes sense to leave this as-is for the sake of code simplicity, even though it does seem like a bit of a waste.